libs/domain/cart/totals/src/totals.component.ts

```diff
@@ -151,7 +151,7 @@ export class CartTotalsComponent extends CartComponentMixin<CartTotalsComponentO
             <small class="delivery-message">
               ${ifDefined(options?.deliveryMessage)}
               <oryx-icon-button size=${Size.Sm}>
-                <a href="#" title=${i18n('cart.totals.delivery-message')}>
+                <a href="#" title=${i18n('cart.totals.not-implemented-yet')}>
```

Comment:
We have an issue with using `i18n` as attribute directive on ssr. Replace content of the title by just a string. It will be enough for now. And pay attention, ticket contains additional requirements: 

`!!! When it is fixed → uncomment line 180 in file apps/storefront-e2e/src/integration/cart.cy.ts and run E2E tests to check if it works again. Also remove the comment in 179`